vue.config.js

```diff
@@ -45,6 +45,20 @@ module.exports = {
     }
   },
   chainWebpack: config => {
+    config.module
+      .rule("fonts")
+      .test(/\.(ttf|otf|eot|woff|woff2)$/)
+      .use("url-loader")
+      .loader("url-loader")
+      .tap(options => {
+        options = {
+          // limit: 10000,
+          name: "./fonts/[name].[ext]",
+          publicPath: process.env.NODE_ENV === "production" ? "./" : "/"
```

Comment:
Is there a reason this needs to be different only for production? I'm concerned this is something we can't actually validate with our test suite since it only happens in production.